<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite thread-count="1" verbose="0" name="Failed suite [Surefire suite]">
  <test name="Surefire test">
    <classes>
      <class name="roi4cio.TestAddProduct"/>
      <class name="roi4cio.TestLoginPage"/>
      <class name="test.ass.product.TestAddProduct"/>
    </classes>
  </test> <!-- Surefire test -->
  <test name="Surefire test(failed)">
    <classes>
      <class name="roi4cio.TestAddProduct">
        <methods>
          <include name="testAddProductConfirmationPopup"/>
          <include name="tearDown"/>
          <include name="setUp"/>
          <include name="testAddProductWithSameName"/>
        </methods>
      </class> <!-- roi4cio.TestAddProduct -->
      <class name="roi4cio.TestLoginPage">
        <methods>
          <include name="tearDown"/>
          <include name="setUp"/>
          <include name="ValidInputDataForlogin"/>
        </methods>
      </class> <!-- roi4cio.TestLoginPage -->
    </classes>
  </test> <!-- Surefire test(failed) -->
</suite> <!-- Failed suite [Surefire suite] -->
